builds/misc/templates/image-windows.yaml

```diff
@@ -8,6 +8,3 @@ parameters:
 steps:
   - powershell: scripts/windows/build/Publish-DockerImage.ps1 -Name "${{ parameters.imageName }}" -Project "${{ parameters.project }}" -Version $(Build.BuildNumber) -Registry $(registry.address) -Namespace "${{ parameters.namespace }}" -Push
     displayName: Build Image - ${{ parameters.name }} - amd64
-  - powershell: scripts/windows/build/Publish-DockerImage.ps1 -Name "${{ parameters.imageName }}" -Project "${{ parameters.project }}" -Version $(Build.BuildNumber) -Registry $(registry.address) -Namespace "${{ parameters.namespace }}" -Architecture "arm32v7" -Push
-    displayName: Build Image - ${{ parameters.name }} - arm32
-    condition: and(succeeded(), eq('${{ parameters.arm32v7 }}', 'true'))
```

Comment:
remove this "parameters.arm32v7" in line 6

Reply:
I already did